app/src/components/main/scrollWrapper/scrollWrapper.scss

```diff
@@ -7,6 +7,12 @@
       opacity: 0.5;
     }
   }
+
+  &.no-scroll {
```

Comment:
Can this be achieved by using `renderTrackVertical` prop?

Reply:
removed